### DISCUSSION TITLE
사이드바 폭 사이즈로인한 애드센스 광고삽입 문제

### DISCUSSION THREAD

**Discussion by moonsunenergy** [pronist/hELLO#138] [Q&A] [answered]
스킨 정말 예쁘고 좋은데 단하나 아쉬운점은 사이드 바의 폭이 50px 정도만 커져도 애드센스 광고를 넣을경우 완성도라던가 보기에도 더 좋아 질것 같습니다.
사이드바 수정하면 전체적으로 문제가 되는 부분이 있을까요?
수정해도 된다면 어느 부분을 하면 되나요?
추후 업데이트버전에서 그러한 부분을 고려해 주시면 안될까 싶습니다.

**Comment by pronist** [comment, marked as answer]
@moonsunenergy 안녕하세요 :)

현재 제 스킨의 방향성이 광고친화적으로 나아가는 것과는 거리가 멀어서
그 패치는 가급적 안 하게 될 가능성이 큰데,

그래도 추후에 인터페이스를 개선하게 될 때 고려는 해보도록 하겠습니다.

의견 감사드립니다. :)
